packages/components/src/context-actions/ContextActions.tsx

```diff
@@ -37,35 +52,64 @@ const log = Log.module('ContextActions');
  * Right clicking the container will then build the context menu, bubbling up until an element with a ContextMenuRoot is on it.
  * You should generally have a ContextMenuRoot on the root node of your document.
  */
-class ContextActions extends Component {
-  static triggerMenu(element, clientX, clientY, actions) {
+class ContextActions extends Component<
+  ContextActionsProps,
+  ContextActionsState
+> {
+  /**
+   * Group you can assign to context menu actions to group them together.
+   * Lower group IDs appear at the top of the list.
+   * Groups are separated by a separator item.
+   * Items within groups are ordered by their order property, then by their title.
+   */
+  static groups = {
```

Comment:
Doesn't necessarily need to be changed now, but we should be using `SCREAMING_SNAKE_CASE` for constants

Reply:
Leaving for now. It's used in the current casing in ~25 places in OSS and probably more in enterprise